docs/earthly-config/earthly-config.md

```diff
@@ -58,6 +58,11 @@ earthly config global.cache_size_mb 20000
 
 Specifies the total size of the BuildKit cache, in MB. The BuildKit daemon uses this setting to configure automatic garbage collection of old cache. A value of 0 causes the size to be adaptive depending on how much space is available on your system. The default is 0.
 
+### cache_size_pct
+
+Similar to [`cache_size_mb`](#cache_size_mb) (including the default and treatment of 0), but expressed as a percentage (0-100) of available space.
```

Comment:
What if we change this to:

```
Specifies the total size of the BuildKit cache, as a percentage (0-100) of the total filesystem size.
When used in combination of `cache_size_mb`, the lesser of the two values will be used. This limit is ignored when set to `0`.
```

I think that might be more clear rather than saying `(including the default and treatment of 0)`.

Reply:
Implemented almost verbatim, except s/of/with/ and removing the `0` markup to stay consistent with the rest of the doc.